### PR TITLE
fix(ONYX-274): update copy for title and sugested artworks modal

### PR DESCRIPTION
**Pull request by tam-kis**
The type of this PR is: FIX

This PR solves [ONYX-274](https://artsyproduct.atlassian.net/browse/ONYX-274)

### Description

Update copy and add artistName to the Suggested Artworks modal title.

Screenshot 2023-08-31 at 09 49 22

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ONYX-274]: https://artsyproduct.atlassian.net/browse/ONYX-274?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/SuggestedArtworksModal.tsx [hunk 1]
```diff
@@ -12,20 +11,17 @@ interface SuggestedArtworksModalProps {
...
+  const { criteria, entity } = useSavedSearchAlertContext()
+  const artistName = entity.defaultCriteria?.artistIDs?.[0].displayValue ?? ""
 
   return (
     <ModalDialog
       width={["100%", 700]}
       onClose={onClose}
-      title={t(
-        "artworkPage.artworkAuctionCreateAlertHeader.suggestedArtworksModal.title"
-      )}
```
Comment (dariakoko) [inline src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/SuggestedArtworksModal.tsx:24]: How do we decide when to add translations?

File: src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/SuggestedArtworksModalHeader.tsx [hunk 2]
```diff
@@ -1,26 +1,33 @@
...
-              key={`filter-label-${pill.field}-${pill.value}`}
-              variant="filter"
-              mb={1}
-              disabled
-            >
-              {pill.displayValue}
-            </Pill>
-          )
-        })}
+    <>
```
Comment (dariakoko) [inline src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/SuggestedArtworksModalHeader.tsx:9]: Do we need this extra wrapper?

**Review by dariakoko — COMMENTED** [review]
Thank you!
I only left two not blocking questions

File: src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/SuggestedArtworksModal.tsx [hunk 3]
```diff
@@ -12,20 +11,17 @@ interface SuggestedArtworksModalProps {
 export const SuggestedArtworksModal: FC<SuggestedArtworksModalProps> = ({
   onClose,
 }) => {
-  const { t } = useTranslation()
-  const { criteria } = useSavedSearchAlertContext()
+  const { criteria, entity } = useSavedSearchAlertContext()
+  const artistName = entity.defaultCriteria?.artistIDs?.[0].displayValue ?? ""
```
Comment (olerichter00) [inline src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/SuggestedArtworksModal.tsx:15]: Hmm, do you know why `artistIDs` is an array of objects and not strings or IDs?

**Review by olerichter00 — APPROVED** [review]
🌟

File: src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/SuggestedArtworksModal.tsx (hunk 1 above)
Comment (tam-kis) [reply src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/SuggestedArtworksModal.tsx:24]: In the meantime it was recommended not to use translations for now, so in this specific case I removed it only for visibility purposes until there is a decision about translations in general.

File: src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/SuggestedArtworksModalHeader.tsx (hunk 2 above)
Comment (tam-kis) [reply src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/SuggestedArtworksModalHeader.tsx:9]: It's a leftover, will remove it 👍

**Review by dariakoko — APPROVED** [review]

**Pull request merged**